mindmaps-migration/owl/src/main/java/io/mindmaps/migration/owl/OWLMigrator.java

```diff
@@ -139,7 +170,7 @@ public Entity entity(OWLNamedIndividual individual) {
                     .findFirst();
             return expr.isPresent() ? expr.get().getClassExpression().asOWLClass() : null;
         });
-        return graph.putEntity(id, owlclass == null ? owlThingEntityType() : entityType(owlclass));
+        return putEntity(id, owlclass == null ? owlThingEntityType() : entityType(owlclass));
```

Comment:
Doesn't `putEntity` end up calling `graph.putEntity` at some point ? If not then don't worry about it. You have already repaired the issue.